new-era-commands/slash/bookmark.js

```diff
@@ -0,0 +1,21 @@
+const { SlashCommandBuilder, EmbedBuilder } = require('discord.js');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('bookmark')
+    .setDescription('Info about bookmarking messages in the TOP Discord server'),
+  execute: async (interaction) => {
+    const bookmarkEmbed = new EmbedBuilder()
+      .setColor('#cc9543')
+      .setTitle('Bookmark messages in this server')
+      .setDescription(`
+To bookmark a message, react with the bookmark icon 🔖 on the message you wish to save for later. 
+
+Our bot will then deliver the message to your DMs in an embedded format with a link to the original message in the server. 
```

Comment:
Instead of `Our bot`, what about pinging the bot directly? You can access bot's user id in the config file. `config.clientId` should be it. Thoughts?

Reply:
Wonderful idea! Implemented the change right away!